cupy/random/_generator.py

```diff
@@ -801,7 +801,7 @@ def seed(self, seed=None):
                 seed = (time.time() * 1000000) % _UINT64_MAX
         else:
             if isinstance(seed, numpy.ndarray):
-                seed = int(hashlib.md5(
+                seed = int(hashlib.sha1(
```

Comment:
Please keep using `md5` here, as this change will (unnecesarily) break reproducibility of existing users with fixed seed.